docs/connect/odbc.md

```
@@ -0,0 +1,159 @@
+rs.Close
+cn.Close
+```
+
+## See also
+
+- {ref}`Turbodbc -- a high-performance ODBC library <turbodbc>`
+
+
+[.NET Framework Data Provider for ODBC]: https://learn.microsoft.com/en-us/dotnet/framework/data/adonet/data-providers#net-framework-data-provider-for-odbc
+[Connecting to PostgreSQL with pyodbc]: https://github.com/mkleehammer/pyodbc/wiki/Connecting-to-PostgreSQL
+[postgres]: https://crates.io/crates/postgres
```

markdownlint-cli2 (0.18.1)

Link and image reference definitions should be needed
Unused link or image reference definition: "postgres"

(MD053, link-image-reference-definitions)